### PR TITLE
lightwallet: Use gray circles & font for informative audits

### PR DESCRIPTION
**Pull request by khempenius**
Screen Shot 2019-05-01 at 12 44 44 AM

**Related Issues/PRs**

### CODE REVIEW

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -438,6 +438,16 @@
 .lh-audit--error .lh-audit__score-icon {
   display: none;
 }
+/* This styling should be listed after pass/average/fail styling for the same elementss, so that it takes precedence */
```
Comment (patrickhulce) [inline lighthouse-core/report/html/report-styles.css]: this makes me think we shouldn't apply the `pass/average/fail` styling when it's informative :)

Comment (khempenius) [reply lighthouse-core/report/html/report-styles.css]: Sounds good to me :) Updated.

File: lighthouse-core/test/report/html/renderer/category-renderer-test.js [hunk 2]
```diff
@@ -96,7 +96,34 @@ describe('CategoryRenderer', () => {
         score: 0,
       },
     });
+    assert.ok(auditDOM.matches('.lh-audit--informative'));
+  });
 
+  it('adds a pass/average/fail class for non-informative audits', () => {
+    const auditDOM = renderer.renderAudit({
+      id: 'informative',
```
Comment (brendankenny) [inline lighthouse-core/test/report/html/renderer/category-renderer-test.js]:
```suggestion
      id: 'not-informative',
```
:)

File: lighthouse-core/test/report/html/renderer/category-renderer-test.js [hunk 3]
```diff
@@ -96,7 +96,34 @@ describe('CategoryRenderer', () => {
...
+      result: {
+        title: 'It informs',
```
Comment (brendankenny) [inline lighthouse-core/test/report/html/renderer/category-renderer-test.js]:
```suggestion
        title: 'It does not inform',
```

**Comment by khempenius** [comment]
Updated.

**Pull request merged**